tslint.json

```diff
@@ -1,14 +1,56 @@
 {
-    "rules": {
-        "no-any": false,
-        "no-empty": true,
-        "no-empty-interface": true,
-        "no-duplicate-imports": true,
-        "no-duplicate-variable": true,
-        "no-unused-variable": true,
-        "triple-equals": true,
-        "prefer-const": true,
-        "arrow-return-shorthand": true,
-        "no-invalid-template-strings": true
-    }
-}
\ No newline at end of file
+  "extends": ["tslint:recommended", "tslint-config-prettier"],
+  "rulesDirectory": ["tslint-plugin-prettier"],
+  "rules": {
+    "prettier": [ true, { "printWidth": 100,
+      "singleQuote": true,
+      "jsxSingleQuote": false,
+      "bracketSpacing": true,
+      "semi": true,
+      "trailingComma": "none",
+      "endOfLine": "crlf" } ],
+    "no-any": false,
+    "no-empty": true,
+    "no-empty-interface": true,
+    "no-duplicate-imports": true,
+    "no-duplicate-variable": true,
+    "triple-equals": true,
+    "prefer-const": true,
+    "no-invalid-template-strings": true,
+    "arrow-return-shorthand": [true, "multiline"],
+    "no-unnecessary-callback-wrapper": true,
+    "typedef": [
+      true,
+      "call-signature",
+      "parameter",
+      "arrow-call-signature",
+      "arrow-parameter",
+      "property-declaration"
+    ],
+    "ordered-imports": true,
+    "object-literal-sort-keys": [
+      true,
+      "ignore-blank-lines",
+      "ignore-case",
+      "locale-compare",
+      "shorthand-first"
+    ],
+    "no-duplicate-super": true,
+    "no-duplicate-switch-case": true,
+    "no-shadowed-variable": true,
+    "no-sparse-arrays": true,
+    "no-switch-case-fall-through": true,
+    "no-tautology-expression": true,
+    "variable-name": {
+      "options": ["check-format", "allow-leading-underscore", "allow-pascal-case"]
+    },
```

Comment:
"sort-keys": ["error", "asc", {"caseSensitive": true, "natural": false, "minKeys": 2}] - add sorting rules for react properties and object keys